### PR TITLE
Added random positions and colors for snakes.

### LINKED ISSUE #1: We need to give the snake a random and UNIQUE color and position
https://github.com/eyal1izhaki/snakes-battle/blob/1d119ba527a580b3b47e54a3a230ec4889914730/snakes_battle/snake.py#L14-L22

### PR DESCRIPTION
**Pull request by davidelkoby7**
Still - not sure how it would handle the length increase after picking up a position for the head.
Do we really want a starting length for the snake? I think just length 1 is best.

### CODE REVIEW

**Comment by eyal1izhaki** [comment]
I think we should give the snakes a random colors from a predefined list because picking a random rgb values can be in some how problematic.
Two colors that looks identical to us can have a different rgb values and colors that are similar to the background color can make it hard to follow the snakes. What do you think?

**Comment by davidelkoby7** [comment]
> 

Sure, you're the leader in this project :)
Didn't merge this PR for a reason? Or can it be merged into dev?
